### ISSUE TITLE
Cannot read properties of undefined (reading 'stop')

### ISSUE THREAD

**Issue by SykesJohn** [rungwiroon/BlazorGoogleMaps#153]
Steps to reproduce:

1. Run the Markers demo in ServerSideDemo.
2. Add a marker.
3. Click on marker.
4. Observe console.

**Comment by valentasm1** [comment]
Which page do you use?
This one mapMarker
Do you click first button "Add marker"?

**Comment by SykesJohn** [comment]
Was that not in the steps to reproduce? I have seen it in more than one place, so it may be everywhere.

**Comment by valentasm1** [comment]
Could not reproduce. Could you provide more info. Exact steps, screenshots, call stack from console.

**Comment by SykesJohn** [comment]
1.  Run ServerSideDemo (Debug mode)
  2.  Click Markers in the menu.
  3.  Click Add marker button to get a marker on the map.
  4.  Click on the marker.
  5.  Browser console shows:
TypeError: Cannot read properties of undefined (reading 'stop')
at Object.invoke (objectManager.js:433)
at blazor.server.js:8
at new Promise (<anonymous>)
at e.beginInvokeJSFromDotNet (blazor.server.js:8)
at blazor.server.js:1
at Array.forEach (<anonymous>)
at e.invokeClientMethod (blazor.server.js:1)
at e.processIncomingData (blazor.server.js:1)
at e.connection.onreceive (blazor.server.js:1)
at WebSocket.i.onmessage (blazor.server.js:1)

If you tell me how to submit (push) my branch to the repository you can have my code (I have added some demos).

**Comment by valentasm1** [comment]
Which browser?
For me windows chrome, opera and firefox works well. 
You could create public repository or fork current on from top right.

**Comment by SykesJohn** [comment]
Chrome. I’ll try Edge.

**Comment by valentasm1** [comment]
Really sorry, but i dont know how i could help you. Maybe you need to update solution

**Comment by SykesJohn** [comment]
Thanks. I tried that wit no effect.
I get a 403 when trying to push my branch.

**Comment by valentasm1** [comment]
You have two solutions: create publish repository or fork it.

**Comment by valentasm1** [comment]
Or third option fork and then make Pull Request

**Comment by SykesJohn** [comment]
Could not reproduce in version 1.1.8.

**Comment by SykesJohn** [comment]
Seems to only happen with builds from Visual Studio 2022.

**Comment by valentasm1** [comment]
Cha cha now we exchanged our views. There is issue with event propagating. Lets say Marker::Click 
I commented one part which it adds event to array, since it didnt made sense. Now it does.
So it add event to array and it should return id of event when it happens when it is MouseEvent, so when you do e.Stop(); it should find that event and trigger Stop. Problem that it returns god know what id and closure event is also god know what. 50 line in js
